src/main/java/Etf/user/UserService.java

```diff
@@ -1,12 +1,99 @@
 package Etf.user;
 
+import Etf.loginUtils.JwtProvider;
+import Etf.loginUtils.SecurityUtils;
+import Etf.user.dto.*;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.util.NoSuchElementException;
+
+@RequiredArgsConstructor
 @Service
 public class UserService {
+
     private final UserRepository userRepository;
+    private final JwtProvider jwtProvider;
+
+    public User findByLoginId(String loginId) {
```

Comment:
이 메서드는 확실한 User만 return하는데 리포지토리 메서드와 구분하기 위해 이름을 아래처럼 바꾸는 것도 괜찮지 않나 싶어요.
```suggestion
    public User getByLoginId(String loginId) {
```

Reply:
네